src/platform/plugins/shared/dashboard/public/dashboard_api/track_panel.ts

```diff
@@ -64,13 +64,29 @@ export function initializeTrackPanel(untilLoaded: (id: string) => Promise<undefi
       if (!id) return;
 
       untilLoaded(id).then(() => {
-        setScrollToPanelId(undefined);
         if (scrollPosition !== undefined) {
           window.scrollTo({ top: scrollPosition, behavior: 'smooth' });
           scrollPosition = undefined;
         } else {
-          panelRef.scrollIntoView({ block: 'start', behavior: 'smooth' });
+          const dashboardTop =
+            document.querySelector('.dashboardContainer')?.getBoundingClientRect().top || 0;
+          const clientBottom = window.innerHeight;
+          const { top: panelTop, bottom: panelBottom } = panelRef.getBoundingClientRect();
+
+          console.log(
```

Comment:
Lets remove console.log

Reply:
Removed in [013211e](https://github.com/elastic/kibana/pull/233226/commits/013211ec4138a24a76722420f575930edec87037)